### PR TITLE
[DOCS] standardizes references to supported Python versions

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
## Description
- updates the supported versions of Python in _data.jsx
- standardizes remaining references to Python versions to use imports.

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

### CODE REVIEW

**Review by Kilo59 — APPROVED** [review]
Thanks for this!

File: docs/docusaurus/docs/components/_data.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export default {
   release_version: 'great_expectations, version 0.17.6',
-  min_python: '3.7',
-  max_python: '3.10'
+  min_python: '3.8',
+  max_python: '3.11'
 }
```
Comment (Kilo59) [inline docs/docusaurus/docs/components/_data.jsx]: what happened to this change?

Comment (kwcanuck) [reply docs/docusaurus/docs/components/_data.jsx]: @Kilo59 it appears that this change was implemented in another PR as it is showing in the file:

image

Comment (Kilo59) [reply docs/docusaurus/docs/components/_data.jsx]: Thanks

**Pull request merged**